src/server/models/Volunteer.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    Volunteer,
+  } from '@/types/volunteer';
+  import { Model, Schema, model, models } from 'mongoose';
+  
+  const VolunteerSchema = new Schema(
+    {
+      firstName: {
+        type: String,
+        required: true,
+      },
+      lastName: {
+        type: String,
+        required: true,
+      },
+      userName: {
+        type: String,
+        required: true,
+      },
+      id: {
```

Comment:
We won't be needing an id here either. Sorry...

Reply:
removed